lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2653,7 +2653,7 @@ def scatter(self, xs, ys, zs=0, zdir='z', s=20, c=None, depthshade=True,
         if np.may_share_memory(zs_orig, zs):  # Avoid unnecessary copies.
             zs = zs.copy()
 
-        patches = super().scatter(xs, ys, s=s, c=c, *args, **kwargs)
```

Comment:
Here and below at `super.bar()`, are we sure that the super class method can't use any `*args`? Don't want to break any plots that rely on it if so.

Reply:
Works for me!